packages/babel-types/src/definitions/experimental.js

```diff
@@ -26,9 +26,16 @@ defineType("AwaitExpression", {
 defineType("BindExpression", {
   visitor: ["object", "callee"],
   aliases: ["Expression"],
-  fields: {
-    // todo
-  },
+  fields: !process.env.BABEL_TYPES_8_BREAKING
```

Comment:
I used `!` everywhere so that, when deleting it for Babel 8, I only have to make one selection (from `!` to `:`) rather than two (from `!` to `?` and then from `:` to the end).

Dev-optimization? :smile: